_notices/rsn0030.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: notice
+parent: RAPIDS Support Notices
+grand_parent: RAPIDS Notices
+nav_exclude: true
+notice_type: rsn
+# Update meta-data for notice
+notice_id: 30 # should match notice number
+notice_pin: true # set to true to pin to notice page
+title: "Docker Image Changes in Release v23.06"
+notice_author: RAPIDS TPM
+notice_status: In Projgess
+notice_status_color: yellow
+# 'notice_status' and 'notice_status_color' combinations:
+#   "Proposal" - "blue"
+#   "Completed" - "green"
+#   "Review" - "purple"
+#   "In Progress" - "yellow"
+#   "Closed" - "red"
+notice_topic: Platform Support Change
+notice_rapids_version: "v23.06+"
+notice_created: 2023-04-20
+# 'notice_updated' should match 'notice_created' until an update is made
+notice_updated: 2023-04-20
+---
+
+## Overview
+
+RAPIDS is making a number of changes to its end-user Docker images to improve functionality, performance, and maintainability. These changes are targeted to be introduced in the RAPIDS 23.06 release.
+
+While many of these changes should be seamless to users, some of them may be breaking. These breaking changes include:
+rapidsai/rapidsai-core is deprecated and superceded by rapidsai/rapidsai and rapidsai/rapidsai-notebooks
+"Runtime" images (eg rapidsai/rapidsai:23.02-cuda11.8-runtime-ubuntu22.04-py3.10) are replaced by the rapidsai/rapidsai-notebooks docker repo
+IMAGE_TYPE will be removed from image tags. Change ${RAPIDS_VER}-cuda${CUDA_VER}-${IMAGE_TYPE}-${LINUX_VER}-py${PYTHON_VER} to ${RAPIDS_VER}-cuda${CUDA_VER}-${LINUX_VER}-py${PYTHON_VER}

```

Comment:
Let's use code font (in backticks) for all code, like the variables here. Applies in several places below.
```suggestion
`IMAGE_TYPE` will be removed from image tags. Change `${RAPIDS_VER}-cuda${CUDA_VER}-${IMAGE_TYPE}-${LINUX_VER}-py${PYTHON_VER}` to `${RAPIDS_VER}-cuda${CUDA_VER}-${LINUX_VER}-py${PYTHON_VER}`
```